### ISSUE TITLE
[Fluxstore App] - Version: 1.6.0---- Unable to run app on ios simulator (Xcode)

### ISSUE THREAD

**Issue by itej7** [inspireui/support#1896]
Hi InspireUI Team,

My issue: We are not able to run this app on ios simulator. We are getting below errors while trying to run on Mac:

Xcode's output:
           ↳
[        ]     /Users/h-mon/.pub-cache/hosted/pub.dartlang.org/firebase_auth-0.15.3/ios/Classes/FLTFirebaseAuthPlugin.m:493:38: error: no known class method for selector 'credentialWithProviderID:IDToken:rawNonce:accessToken:'
                     credential = [FIROAuthProvider credentialWithProviderID:provider
                                                    ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
               /Users/h-mon/.pub-cache/hosted/pub.dartlang.org/firebase_auth-0.15.3/ios/Classes/FLTFirebaseAuthPlugin.m:502:38: error: no known class method for selector 'credentialWithProviderID:IDToken:rawNonce:'
                     credential = [FIROAuthProvider credentialWithProviderID:provider
                                                    ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
               2 errors generated.
               : note: 'timestampsInSnapshotsEnabled' has been explicitly marked deprecated here
                   __attribute__((deprecated));
                                  ^
               1 warning and 4 errors generated.
               note: Using new build system
               note: Planning build
               note: Constructing build description
               warning: Mapping architecture armv7 to i386. Ensure that this target's Architectures and Valid Architectures build settings are configured correctly for the iOS Simulator platform. (in target 'image_picker' from project 'Pods')
               warning: Mapping architecture arm64 to x86_64. Ensure that this target's Architectures and Valid Architectures build settings are configured correctly for the iOS Simulator platform. (in target 'image_picker' from project 'Pods')

[   +6 ms] Could not build the application for the simulator.
[   +2 ms] Error launching application on iPhone 11 Pro Max.

**screenrecorder output of issue:**

![image](https://user-images.githubusercontent.com/50324759/72796565-8ccf9800-3c65-11ea-964a-1692c5775489.png)

Proof of my purchase:
Product Purchase code: c2f1abaf-6ebe-4c07-875d-050022ba2a14 - 14 Oct 2019 REGULAR LICENSE

**Comment by minhcasi** [comment]
@itej7 - pls update to latest 1.6.2 - thank you!

**Comment by itej7** [comment]
@minhcasi : Can you assist us by taking remote or by going through our source code as we have fully customized this app and already launched it on playstore. Kindly assist us as we can not afford to update it and start the customization from zeroth position again.

**Comment by minhcasi** [comment]
@itej7 - pls try following solution to clean cache
- run "flutter clean"
- remove .symlinks hidden folder from
- remove Pods folder, Podfile, and Podfile.lock

Then press run button again on Android studio

**Comment by itej7** [comment]
@minhcasi : Thanks for the reply. We will check and update you accordingly.

**Comment by itej7** [comment]
@minhcasi : We are still not able to run this on ios simulator.
Kindly assist by taking remote of our mac system. I have also messaged you on skype.
Please reply us on skype.

**Comment by minhtruong315** [comment]
Pls try with latest 1.6.3 as the old one is not support for latest Flutter that cause previous issues, as your app is customize pls try to manually upgrade Flutter by your own time - we may not support for heavy customize that could cause bug and issues, hope you can understand

**Comment by itej7** [comment]
@minhtruong315

> Pls try with latest 1.6.3 as the old one is not support for latest Flutter that cause previous issues, as your app is customize pls try to manually upgrade Flutter by your own time - we may not support for heavy customize that could cause bug and issues, hope you can understand

@minhtruong315 we have recently updated the entire app due to same communication from your developer ...when you say we have customized the app but in reality we have only changed the color scheme and bit of placement without touching any core customization.
stating this the android version is working smoothly we are only facing the issue on iOS.
since we started the development on your app it has been revised twice, to resolve small issues you are asking customer to update entire code base which is not feasible in real world.

We cannot keep changing the entire codebase now and then without even launching the app.

Request you to assist us to get this issue resolved as soon as possible so we can close this issue, we have been very patient for this app update but we simply cannot update the entire code base this time.
